worker/include/RTC/IceServer.hpp

```diff
@@ -61,15 +61,15 @@ namespace RTC
 		{
 			return this->selectedTuple;
 		}
-		void SetUsernameFragment(const std::string& usernameFragment)
+		void RestartIce(const std::string& usernameFragment, const std::string& password)
```

Comment:
Why is this method renamed?

Reply:
The only usage of `SetUsernamFragment ` and `SetPassword` is when restarting ice.
And remoteNomination also need to be set to 0 at the same time.
So instead of adding another function like `SetNomination`, i perferred to merge them into one.
`RestartIce` is an action and we can do more thing in it.